draft-packages/avatar/KaizenDraft/Avatar/styles.module.scss

```diff
@@ -0,0 +1,96 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/shadow";
+@import "~@kaizen/design-tokens/sass/typography";
+
+.wrapper {
+  background: $kz-color-stone;
+  border-radius: 100%;
+  overflow: hidden;
+  align-items: center;
+  display: flex;
+  justify-content: center;
+
+  &:not(.small) {
+    box-shadow: $kz-shadow-small-box-shadow;
+    border: 3px solid $kz-color-white;
+  }
+  // Themes
+
+  &.personal {
+    background: $kz-color-peach-200;
+  }
+
+  &.otherUser {
+    background: $kz-color-wisteria-200;
+  }
+
+  // Sizes
+
+  &.small {
+    width: 1.25rem;
+    height: 1.25rem;
```

Comment:
Suggest doing these in multiples of `$ca-grid` – that's the pattern that's encouraged in Kaizen. Makes it easier to reason about.

You can get it from `@import "~@kaizen/component-library/styles/grid";`.

Reply:
Sweet, I thought that might have been exclusively for padding / margin but i'll update these